pokecli.py

```diff
@@ -213,6 +213,7 @@ def get_commit_hash():
                 )
                 time.sleep(36000)
             except NoPlayerPositionSetException:
+                wait_time = config.reconnecting_timeout * 60
```

Comment:
We need to move the scope of this. We use it further down the line with different excepts.